advisories/published/2025/JLSEC-0000-mntg6n3ny-pvwevf.md

```diff
@@ -0,0 +1,24 @@
+```toml
+schema_version = "1.7.4"
+id = "JLSEC-0000-mntg6n3ny-pvwevf"
+modified = 2025-11-20T23:04:00.334Z
+upstream = ["CVE-2021-45451"]
+references = ["https://github.com/ARMmbed/mbedtls/releases/tag/v3.1.0", "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/IL66WKJGXY5AXMTFE7QDMGL3RIBD6PX5/", "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/TALJHOYAYSUJTLN6BYGLO4YJGNZUY74W/", "https://github.com/ARMmbed/mbedtls/releases/tag/v3.1.0", "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/IL66WKJGXY5AXMTFE7QDMGL3RIBD6PX5/", "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/TALJHOYAYSUJTLN6BYGLO4YJGNZUY74W/"]
+
+[[affected]]
+pkg = "MbedTLS_jll"
+ranges = ["*"]
```

Comment:
This is another misreading. [mbedtls says](https://web.archive.org/web/20220303224544/https://tls.mbed.org/tech-updates/security-advisories/mbedtls-security-advisory-2021-12):

> Affected users will want to upgrade to Mbed TLS 3.1.0, 2.28.0 or 2.16.12 depending on the branch they're currently using.

```suggestion
ranges = ["< 2.28.0+0"]
```